Content.Shared/GameObjects/Verbs/GlobalVerb.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Robust.Shared.Interfaces.GameObjects;
+
+namespace Content.Shared.GameObjects
+{
+    /// <summary>
+    ///     A verb is an action in the right click menu of an entity.
+    ///     Global verbs are visible on all entities, regardless of their components.
+    /// </summary>
+    /// <remarks>
+    ///     To add a global verb to all entities,
+    ///     define it and mark it with <see cref="GlobalVerbAttribute"/>
+    /// </remarks>
+    [UsedImplicitly]
+    public abstract class GlobalVerb
+    {
+        /// <summary>
+        ///     If true, this verb requires the user to be inside within
+        ///     <see cref="InteractionRange"/> meters from the entity on which this verb resides.
+        /// </summary>
+        public virtual bool RequireInteractionRange => true;
+
+        public const float InteractionRange = 2;
```

Comment:
You should put these constants somewhere else so they can be shared with regular `Verb`.

Reply:
Moved `InteractionRange` and `InteractionRangeSquared` into `VerbUtility`. Kept `RequireInteractionRange` in each verb class since it's meant to be overriden by verb implementations.